servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/DefaultDnsServiceDiscovererBuilder.java

```diff
@@ -62,6 +62,7 @@ public final class DefaultDnsServiceDiscovererBuilder implements DnsServiceDisco
     private static final SocketAddress DEFAULT_LOCAL_ADDRESS =
             getBoolean(SKIP_BINDING_PROPERTY) ? null : new InetSocketAddress(0);
     private static final DnsResolverAddressTypes DEFAULT_DNS_RESOLVER_ADDRESS_TYPES = systemDefault();
+    private static final int DEFAULT_CONSOLIDATE_CACHE_SIZE = 1024;
```

Comment:
Is there any reasoning how we ended up with 1024 as the cache size number?

Reply:
Not really, just a number that feels not too big and not too low, handling most of the use-cases that feel reasonable (having up to this number of unique hostnames). Happy to adjust if there are other proposals

Reply:
No counter proposal, sounds reasonable to me!